routemaster.rb

```diff
@@ -2,4 +2,28 @@ module Routemaster
   def self.now
     (Time.now.utc.to_f * 1e3).to_i
   end
+
+  def self.configure(**options)
+    @_config = options
+  end
+
+  def self.config
+    {
+      redis_pool_size: 1
+    }.merge(@_config || {})
```

Comment:
Would it be worth memoizing this so there isn't a hash allocation each time it's called? (Or alternatively doing the merge in the `configure` method?